quart_2/seminar_4/task_1.py

```diff
@@ -0,0 +1,11 @@
+# 1. Вычислить число c заданной точностью d
+# Пример:
+# при d = 0.001, π = 3.142 10^(-1) ≤ d ≤ 10^(-10)
+
+from math import pi
+
+d =  float(input("Введите число для заданной точности числа π: "))
+
+count_after_dot = len(str(d).split('.')[1])
+
+print(f'при = {d}, π = {round(pi, int(count_after_dot))}')
```

Comment:
```suggestion
print(f'при = {d}, π = {round(pi, count_after_dot)}')
```
Функция `len` и так возвращается целое число.